pkg/controller/perconaservermongodb/psmdb_controller.go

```diff
@@ -250,7 +250,25 @@ func (r *ReconcilePerconaServerMongoDB) Reconcile(request reconcile.Request) (re
 			return reconcile.Result{}, err
 		}
 
-		// Create Service
```

Comment:
just Q: why it was needed to move?

Reply:
Not needed, but I felt it would make more sense to move creation of the headless service up here because it is now always done while the external service creation is still conditional. Don't mind to move it down if you prefer it.